assets/babel-plugin.js

```diff
@@ -50,6 +50,13 @@ function rewriteImport(importMap, imp, dir, shouldAddMissingExtension) {
   if (isSourceImport && shouldAddMissingExtension && !path.extname(imp)) {
```

Comment:
I don't think that this option will work with `shouldAddMissingExtension`, since it will get caught in this if-block above and just returned as `../svg` -> `../svg.js`.